neon_llm_palm2/palm2.py

```diff
@@ -150,7 +164,8 @@ def _embeddings(self, question: str, answers: List[str], persona: dict) -> (List
         """
         response = self.ask(question, [], persona=persona)
         texts = [response] + answers
-        embeddings = get_embeddings(texts, engine="text-embedding-ada-002")
+        embeddings_obj = self._embedding.get_embeddings(texts)
```

Comment:
Should this reference `self.embedding` to ensure `self._embedding` is defined?

Reply:
Done with https://github.com/NeonGeckoCom/neon-llm-palm2/pull/1/commits/d3e3e06aca7a2af53e77ef697a3fa8e03cbf3a3a